malaria-ios/ViewControllers/PlanTrip/PlanTripViewController.swift

```diff
@@ -125,7 +154,13 @@ extension PlanTripViewController{
     }
     
     @IBAction func locationEditingChangedHandler(sender: AnyObject) {
-        updateLocation(location.text!)
+        //updateLocation(location.text!)
+        if let loc = location.text {
+            generateTripBtn.enabled = !loc.isEmpty
+            print("3. generateTripBtn.enabled", generateTripBtn.enabled)
```

Comment:
This kind of logs can be removed. Use Logger.Info instead in those cases (try to see the difference :) )